src/components/StripeCardInput/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import scriptLoader from 'react-async-script-loader';
+import CardInput from './component';
+
+const stripeApiV3Url = 'https://js.stripe.com/v3';
+
+/**
+ * Stripe Card Input component are used for freeform data entry.
+ * @category Form
+ */
+const StripeCardInput = props => {
+    const { ...rest } = props;
+
+    const Component = useCallback(scriptLoader(stripeApiV3Url)(CardInput), []);
```

Comment:
if it doesn't depend on any props why is inside of the component, it could be another `const`, right?